### DISCUSSION TITLE
Replace "copyright" syntax with proper URL syntax

### DISCUSSION THREAD

**Discussion by mjog** [analog-garage/mkdocstrings-python-xref#25] [Ideas]
Was super chuffed to find this package, because absolute refs are a pain, but was wondering about the the use of `(c)` etc instead of (say) URI schemes. Since mkdocstring's `[][]` syntax is a riff of Markdown's standard link syntax, mkdocstrings-python-xref's copyright-symbol-style looks *really weird* from my POV, as someone who has written a lot of Markdown.

Would you be open to using URL schemes instead? Like `cls:` instead of `(c)` or `pkg:` instead of `(p)`?

I feel like `cls:__init__` is much more obvious than `(c).__init__`, likewise for `mod:MyClass.some_attr` instead of `(m)MyClass.some_attr`.

**Comment by mjog** [comment]
That would then open the door to using the standard, well understood, URL-path-like semantics rather than things like `^` and `^^` which to be honest I'd like to use, but probably won't ever since I'd have to look them up every time I do to remember what they mean.

**Comment by analog-cbarber** [comment]
These aren't URLS. They are just relative references to python qualified references. Even in Markdown the ref in the `[][]` syntax was always meant to be a local bookmark not a full URL (which uses the `[]()` syntax).

I would be open to extending the syntax but only if there many people who want this. Otherwise, it is not worth the trouble of adding complexity to the syntax and to the implementation.
